components/proxy/conf/Caddyfile

```diff
@@ -223,6 +223,16 @@ https://{$GITPOD_DOMAIN} {
 		}
 	}
 
+	@local_app path /static/bin/*
```

Comment:
We should not server all static files from ide proxy, but only existing links, please make it specific.

Reply:
no all static files, just path start with /static/bin/
After I checked, only local-app is currently using this directory, but I can change it as you said

Reply:
@akosyakov I already changed it according to your suggestion